product_docs/docs/odbc_connector/13/04_creating_a_data_source.mdx

```diff
@@ -27,8 +27,6 @@ Password = manager
 Port = 5444
 ```
 
-The first line in the data source is the data source name. The name is a unique identifier, enclosed in square brackets. The data source name is followed by a series of `'keyword=value'` pairs that identify individual connection properties that make up the data source.
+The first line in the data source is the data source name. The name is a unique identifier, enclosed in square brackets. The data source name is followed by a series of `keyword=value` pairs that identify individual connection properties that make up the data source.
 
-The ODBC administrator utility creates named data sources for ODBC connections. In most cases, an ODBC administrator utility is distributed with the operating system (if you’re using Windows or unixODBC, the tool is called the `ODBC Data Source Administrator`). If your operating system doesn’t include an ODBC administrator, third-party options are available online.
-
-Sections `Adding a Data Source Definition in Windows` and `Adding a Data Source Definition in Linux` walk you through adding a data source in Windows and Linux using the graphical tools available for each operating system. During the process of defining a data source, you’ll be asked to specify a set of connection properties. Section `EDB-ODBC Connection Properties` contains information about `optional` data source connection properties; you can specify connection properties with graphical tools or edit the `odbc.ini` file with a text editor.
```

Comment:
@ebgitelman did you intentionally delete this paragraph? I think we should keep it.

Reply:
I'll have to double check that those sections actually exist.

Reply:
@drothery-edb I added this back in and inserted proper links so this should be ready to merge.